stories/topics/ref-gcp-supported-changes.adoc

```diff
@@ -11,18 +11,20 @@ Red Hat supports changes to the following options:
 * DNS and `resolv.conf` files
 
 .Red Hat Responsibilities
-Red Hat has the following responsibilities.
 
-* Premium support for {PlatformNameShort}.
+Red Hat has the following responsibilities:
+
+* Premium support for {PlatformNameShort}
 * Directions and how-to processes for upgrading {AAPonGCP}.
-** Provided through knowledge base articles when upgrades are published.
+* Red Hat will support the current version of {PlatformNameShort}. Bug fixes and CVE patches will require an upgrade to the latest version
```

Comment:
Red Hat supports the current version of {PlatformNameShort}   

(does this mean previous versions are not supported?)

Bug fixes and CVE patches require an upgrade to the latest version.

Reply:
This is meaning that bug fixes and CVE patches are fixed in the next released version of Self managed AoC